### ISSUE TITLE
Specify arbitrary attributes in fenced code (make SuperFences more JS highlight friendly)

### ISSUE THREAD

**Issue by facelessuser** [facelessuser/pymdown-extensions#895]
In SuperFences, you can specify arbitrary class names and ids, but you cannot specify general attributes. This is partly because we reserve the format `key="value"` for options. But what if, following existing selector style for classes and ids, we allowed for `[attr="value"]`. This would distinguish it from options and use a familiar pattern. Yeah it doesn't match `attr_list`, but 🤷‍♂️.

So this would allow people to do things like the following (useful for JS highlighters):

````
```{py .line-numbers [data-start="-5"] some-option="whatever"}
import some_module

# Do something
```
````

As a separate issue, it may be nice to configure `linenums` option to allow the user to specify a class name.

**Comment by facelessuser** [comment]
I think I went down this road before and decided that with custom fences, you could add options that could set attributes. Maybe we will just add option to specify linenums class.

**Comment by facelessuser** [comment]
Specifying linenums class is solved by #901

**Comment by facelessuser** [comment]
I think we will leave this as is for now. We've added an option to allow specifying the line number class. If desired, a user can create custom fences if they wish to allow setting specific attributes
